ios/ci.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+# This script runs from its own folder
+cd "$(dirname "$THIS_SCRIPT")"
+
+# Include our resource functions; they're pretty useful!
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# Please note that this build script (understandably) assumes that it is running on Mac OS X.
+verify_on_mac
+
+builder_describe "Defines actions for use in CI-related iOS builds." \
+  "build" \
+  "prep-release"
+  # For consideration:  clean               ("Cleanup Carthage Mess", "Clean XCode DerivedData Mess")
+  # For consideration:  publish:testflight  ("Upload to TestFlight (PR area)")
+  # For consideration:  publish:app-store   ("Push * to the App Store for Deployment")
+  # For consideration:  publish:fastlane    (for matching release-build steps)
+
+builder_parse "$@"
+
+function do_build() {
+  ./build.sh                         clean configure build --sim-artifact
```

Comment:
```suggestion
  ./build.sh clean configure build --sim-artifact
```

Reply:
As formatted, the actions and options align perfectly with the `../oem/firstvoices/ios/build.sh` call below.  That's the reason for the extra whitespace.

Reply:
Fair enough

Reply:
That's helpful documentation @jahorton, thanks.

Reply:
If there's somewhere specific I should aim to stick this documentation, past the script's definition section themselves, let me know.  It sounds like there's a desire for it... but I'm not clear on _where_.

Reply:
Just stick it in ios/readme.md